Sources/LanguageServerProtocol/SupportTypes/ClientCapabilities.swift

```diff
@@ -465,6 +465,31 @@ public struct TextDocumentClientCapabilities: Hashable, Codable, Sendable {
     }
   }
 
+  public struct CodeLens: Hashable, Codable, Sendable {
+
+    /// Whether the client supports dynamic registration of this request.
+    public var dynamicRegistration: Bool?
+
+    public var supportedCommands: [String: String]
```

Comment:
Could you add a doc comment here? Essentially just paraphrasing https://github.com/swiftlang/sourcekit-lsp/pull/1556#issuecomment-2233373503.